### PR TITLE
solved problem 3.3 from the site stepik to move the robot

### PR DESCRIPTION
**Pull request by Yurii666**

### CODE REVIEW

File: src/Stepic/RobotDemo.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+     public enum Direction {
+         UP,
+         DOWN,
+         LEFT,
+         RIGHT
+     }
+
+     public static class Robot {
+             int x;
```
Comment (Yurii-Loienko) [inline src/Stepic/RobotDemo.java:18]: I don't understand what does it mean - 'x' and what should it do? 
What is the value of this variable?
This is the coordinateX or something else?

File: src/Stepic/RobotDemo.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+             int y;
+             Direction dir;
+
+         public Robot (int x, int y, Direction dir) {
+              this.x = x;
+              this.y = y;
```
Comment (Yurii-Loienko) [inline src/Stepic/RobotDemo.java:24]: the same as above

File: src/Stepic/RobotDemo.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+         public int getX() {return x;}
+
+         public int getY() {return y;}
+
+         public void turnLeft() {
+             if      (dir == Direction.UP)    {dir = Direction.LEFT;}
+             else if (dir == Direction.DOWN)  {dir = Direction.RIGHT;}
+             else if (dir == Direction.LEFT)  {dir = Direction.DOWN;}
+             else if (dir == Direction.RIGHT) {dir = Direction.UP;}
```
Comment (Yurii-Loienko) [inline src/Stepic/RobotDemo.java:39]: Use brackets anywhere, anytime

File: src/Stepic/RobotDemo.java [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+         public void turnRight() {
+             if      (dir == Direction.UP)    {dir = Direction.RIGHT;}
+             else if (dir == Direction.DOWN)  {dir = Direction.LEFT;}
+             else if (dir == Direction.LEFT)  {dir = Direction.UP;}
+             else if (dir == Direction.RIGHT) {dir = Direction.DOWN;}
+         }
+
+         public void stepForward() {
+             if (dir == Direction.UP)    {y++;}
```
Comment (Yurii-Loienko) [inline src/Stepic/RobotDemo.java:50]: the same as above

**Review by Yurii-Loienko — CHANGES_REQUESTED** [review]

File: src/Stepic/RobotDemo.java (hunk 3 above)
Comment (Yurii666) [reply src/Stepic/RobotDemo.java:39]: is it sarcasm, or a message to action?

File: src/Stepic/RobotDemo.java (hunk 4 above)
Comment (Yurii666) [reply src/Stepic/RobotDemo.java:50]: они же здесь и так есть

File: src/Stepic/RobotDemo.java (hunk 3 above)
Comment (Yurii-Loienko) [reply src/Stepic/RobotDemo.java:39]: This is a message to action

**Review by Yurii-Loienko — APPROVED** [review]

**Pull request merged**